src/elementary/IsotopeID/test/IsotopeID.test.cpp

```diff
@@ -51,13 +57,31 @@ SCENARIO( "IsotopeID" ) {
       CHECK( 0 == id.mass() );
       CHECK( "H0" == id.symbol() );
       CHECK( "H0" == id.name() );
+
+      id = IsotopeID( "h0" );
+      CHECK( ElementID( 1 ) == id.element() );
+      CHECK( 0 == id.mass() );
+      CHECK( "H0" == id.symbol() );
+      CHECK( "H0" == id.name() );
+
+      id = IsotopeID( "H" );
+      CHECK( ElementID( 1 ) == id.element() );
+      CHECK( 0 == id.mass() );
+      CHECK( "H0" == id.symbol() );

```

Comment:
I thought that `H` meant hydrogen (the element) and `h` meant hydrogen (an isotope). It seems with this Pull Request, this definition has changed somewhat. Am I mistaken?

Reply:
Yes, I changed that following our discussion from a few days ago. As long as the final symbol has the 0, it should be fine. It was in the pull request description though ;-)